public/locales/fr.json

```diff
@@ -88,6 +88,8 @@
     "displayImage": "Afficher l'image",
     "enterName": "Entrez le nom",
     "sort": "Trier",
+    "Oldest": "Les plus anciennes d'abord",
```

Comment:
1. Use the translation for `latest` and `earliest` in all cases
2. Verify that the translations don't negatively affect the UI/UX
3. Include a video of all the translations in action before the next review